xgcm/test/test_grid.py

```diff
@@ -691,8 +698,8 @@ def test_grid_ops(all_datasets):
         "fill",
         pytest.param("extrapolate", marks=pytest.mark.xfail(strict=True)),
         "extend",
-        {"X": "fill", "Y": "extend"},
-        {"X": "extend", "Y": "fill"},
+        {"X": "fill", "Y": "extend", "Z": "extend"},
```

Comment:
Why is it necessary to add an additional axis here?

Reply:
I think all of the current test cases are 2D, but I added a couple of 3D SGRID tests (one 3D grid, and one 3D grid plus vertical coordinate) It failed here because it's only set for 2D. Just changed similarly in my current work.

Reply:
Removed with removing sgrid datasets from grid tests.